sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the Key Vault Administration client.
+ *
+ */
+
+#pragma once
+#include "settings_client_options.hpp"
+#include <azure/core/context.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/internal/http/pipeline.hpp>
+#include <azure/core/response.hpp>
+#include <azure/keyvault/administration/rest_client.hpp>
+#include <memory>
+#include <string>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Administration {
+#if defined(TESTING_BUILD)
+
+  namespace Test {
+    class SettingsClientTest;
+  }
+#endif
```

Comment:
This should be isolated to a _test.hpp header.  Why does it need to be in the public header?

Reply:
removed, not needed anymore